compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/GraphqlSourceModifier.java

```diff
@@ -445,10 +443,13 @@ private SeparatedNodeList<MappingFieldNode> getUpdatedFields(MappingConstructorE
         List<Node> fields = new ArrayList<>();
         SeparatedNodeList<MappingFieldNode> existingFields = annotationValue.fields();
         Token separator = NodeFactory.createToken(SyntaxKind.COMMA_TOKEN);
-        for (MappingFieldNode field : existingFields) {
-            fields.add(field);
-            fields.add(separator);
+        for (int i = 0; i < existingFields.size(); i++) {
+            fields.add(existingFields.get(i));
+            if (i < existingFields.size() - 1) {
+                fields.add(existingFields.getSeparator(i));
+            }
         }
+        fields.add(separator);
```

Comment:
if `existingFields.size()` is zero, then this separator will add in the first ryt?

Reply:
Having `0` fields in the annotation does not make sense, right?